CHANGELOG.md

```diff
@@ -0,0 +1,105 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+ - API: New function to decode the image using a callback outputting a part of a
+   row per call.
+ - API: 16-bit float output support.
+
+### Changed
+ - Full specification support: Implemented all the spec, including cases that
```

Comment:
Might want to clarify this is about the codestream spec (part 1). There are some things in part 2 (in particular, the `brob`) that are not yet implemented. Not needed to decode to pixels, but can be needed to reconstruct a jpeg file.

Reply:
wdyt now?

Reply:
LGTM.